internal/driver/driver_test.go

```diff
@@ -96,6 +96,7 @@ func TestParse(t *testing.T) {
 		{"peek=line.*01", "cpu"},
 		{"weblist=line[13],addresses,flat", "cpu"},
 		{"tags,tagfocus=400kb:", "heap_request"},
+		{"tags,tagfocus=+400kb:", "heap_request"},
```

Comment:
As I said, add a test for negative values as well?

Reply:
man you are catching all my laziness.  That requires me understanding how you have rigged up all these which is more than a bit of logic...

Reply:
nope, as noted doing that requires understanding your tests.  The current test is a large integration test so it is not particularly amenable to understanding.  The code is simple enough and the positive case hits the same code path, so I strongly prefer to stop there.

Reply:
Sorry, I can't merge this without a test.

Reply:
the test provided hits the correct path.  What you are asking is essentially to test if go's parse int routine handles negative signs.  (it does)

Reply:
There may be other reasons the code may not work nor or with future refactorings, other than parsing. For example, there could be some unsigned types along the path of handling the numeric tag filtering, as an example.